src/roiextractors/extractorlist.py

```diff
@@ -7,7 +8,9 @@
 from .extractors.nwbextractors import NwbImagingExtractor, NwbSegmentationExtractor
 from .extractors.schnitzerextractor import (
     CnmfeSegmentationExtractor,
+    LegacyExtractSegmentationExtractor,
     ExtractSegmentationExtractor,
+    NewExtractSegmentationExtractor,
```

Comment:
Just leaving this as a note - I believe  the intention here is to allow people the option of manually overriding the choice of which sub type to use for their data in the event that the main `ExtractSegmentationExtractor` routing function does not behave as they expect?

Reply:
Yes, it was my intention, on the other hand maybe but if this is confusing to have 3 types of classes of the same type?

Reply:
At first it was confusing, yes, but I can see the need for it. Alternative is to force them to directly import it from the deeper level in that event.

IDK what do you think? I could see a user looking at it and wondering which class they are meant to use for their data

Reply:
having confusing imports is not good probably, the `ExctractSegmentationExtractor` should handle cases when someone wanted to use the older version